packages/astroport/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "astroport"
-version = "2.1.0"
+version = "2.1.1"
```

```suggestion
version = "2.2.0"
```
because functionality was added